app/views/admin/tags/_form.html.erb

```diff
@@ -48,20 +48,12 @@
     <%= f.label :wiki_body_markdown, class: "crayons-field__label" %>
     <%= f.text_area :wiki_body_markdown, value: tag.wiki_body_markdown, class: "crayons-textfield" %>
   </div>
-  <div class="crayons-notice crayons-notice--warning mt-3">We will soon update how tags are being displayed. Please define tag's primary color in "Bg color hex" field - the other one ("Text color hex") will be deprecated.</div>
   <div class="crayons-field mt-3">
-    <%= f.label :bg_color_hex, class: "crayons-field__label" %>
+    <%= f.label :bg_color_hex, "Tag color", class: "crayons-field__label" %>
```

Comment:
Tag color is hard-coded here because we haven't started i18n work in the Admin section.